egs2/cmu_arctic/tts1/conf/tuning/finetune_transformer.yaml

```diff
@@ -0,0 +1,91 @@
+# This configuration is for ESPnet2 to train Transformer-TTS. Compared to the
+# original paper, this configuration additionally use the guided attention
+# loss to accelerate the learning of the diagonal attention. It requires
+# 4 GPUs with 32 GB memory and it will finish within 1 day to finish the
+# training on Tesla V100.
+
```

Comment:
If you add a comment about the modification for fine-tuning, that is helpful.

Reply:
Added a comment about finetune modification.